conda/cli/main.py

```diff
@@ -69,7 +69,16 @@ def _main(*args, **kwargs):
         args = args + ('-h',)
 
     p = generate_parser()
-    args = p.parse_args(args[1:])
+    args, unknown_args = p.parse_known_args(args[1:])
+    if unknown_args:
+        if args.cmd == "run":
+            args.unknown_args = unknown_args
```

Comment:
If you use `argparse.REMAINDER`, you can leave out the the `args.cmd == "run"`-specific part.